docs/build/identity.rs/1.3/docs/references/specifications/revocation-timeframe-2024.mdx

```diff
@@ -0,0 +1,149 @@
+| `id`                    | The constraints on the `id` property are listed in the [Verifiable Credentials Data Model specification](https://www.w3.org/TR/vc-data-model/). The `id` MUST be a [DID URL](https://www.w3.org/TR/did-core/#did-url-syntax) that is the URL to a [Revocation Bitmap Service](#revocation-bitmap-service) in the DID Document of the issuer. |
+| `type`                  | The `type` property MUST be `"RevocationBitmap2022"`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `revocationBitmapIndex` | The `revocationBitmapIndex` property MUST be an unsigned, 32-bit integer expressed as a string. This is the index of the credential in the issuers revocation bitmap. Each index SHOULD be unique among all credentials linking to the same [Revocation Bitmap Service](#revocation-bitmap-service).                                                                                                                                                                                                                                                                                                                          |
+| `startValidityTimeframe`| The `startValidityTimeframe` property MUST be a [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339)-compliant timestamp. |
+| `endValidityTimeframe`| The `endValidityTimeframe` property MUST be a [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339)-compliant timestamp. |
+
+#### Example
+
+An example of a verifiable credential with a `credentialStatus` of type `RevocationTimeframe2024`.
+
+```json
+{
+  "@context": "https://www.w3.org/2018/credentials/v1",
+  "id": "https://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "credentialSubject": {
+    "id": "did:iota:B8DucnzULJ9E8cmaReYoePU2b7UKE9WKxyEVov8tQA7H",
+    "GPA": "4.0",
+    "degree": "Bachelor of Science and Arts",
+    "name": "Alice"
+  },
+  "issuer": "did:iota:EvaQhPXXsJsGgxSXGhZGMCvTt63KuAFtaGThx6a5nSpw",
+  "issuanceDate": "2022-06-13T08:04:36Z",
+  "credentialStatus": {
+    "id": "did:iota:EvaQhPXXsJsGgxSXGhZGMCvTt63KuAFtaGThx6a5nSpw?index=5#revocation",
```

Comment:
```suggestion
    "id": "did:iota:EvaQhPXXsJsGgxSXGhZGMCvTt63KuAFtaGThx6a5nSpw#my-revocation-service",
```

Reply:
The change reflects what has been done in the `10_zkp_revocation` example. Note that the index MUST not be disclosed.